packages/bot-skeleton/src/scratch/hooks/css.js

```diff
@@ -633,12 +634,13 @@ Blockly.Css.CONTENT = [
 
     '.blocklyMainBackground {',
     'stroke-width: 0;',
+    'fill: var(--general-main-1) !important',
     // 'stroke: #c6c6c6;',  /* Equates to #ddd due to border being off-pixel. */
     '}',
 
     '.blocklyMutatorBackground {',
     'fill: #fff;',
-    'stroke: #ddd;',
+    'stroke: #fff;',
```

Comment:
can we use `$color-white` here?

Reply:
in this hooks as you see even in the previous implementation it seems it is not reading the color variables ... when I use the variable it is not getting the color ... lemme knwo if you have a solution for it in the scope of this card

Reply:
I think the reason why the others are working fine, like `--general-main-1` `--text-prominent` is it is directly declared in `app/packages/core/src/index.html` can you check try `--general-main-1` for white colors?

Reply:
I need a const color not a color that changes with theme. so in this case I cannot use general-main-1.